runHarness/Utils.pm

```diff
@@ -255,6 +255,43 @@ sub callBooleanMethodOnObjectsParallel2 {
 	return $retval;
 }
 
+sub callBooleanMethodOnObjectsParallel2BatchDelay {
+	my ( $method, $objectsRef, $param1, $param2, $batchSize, $delaySec ) = @_;
+	my $console_logger = get_logger("Console");
+	my @pids;
+	my $pid;
+	my $objectNum = 0;
+
+	foreach my $object (@$objectsRef) {
+		if ($objectNum == $batchSize) {
+			sleep($delaySec);
```

Comment:
I suggest printing a log message (perhaps debug) when a sleep occurs.
Here in 2 param and in 3 param also.

Reply:
Added a debug log message for now.